spec/features/activity_page_spec.rb

```diff
@@ -62,7 +62,7 @@
       it 'filters the courses to my courses' do
         # Admin is admin of course 1, should only see user1's revision
         # when checked
-        click_link 'Recent Activity'
+        visit '/recent-activity'
```

Comment:
I could totally see this going either way, and I'm sure your reasoning is good for doing this. One thing I like about the `click_link` approach is that, if that link accidentally gets deleted, this covers that. Of course, one annoying thing is if you change it from 'Recent Activity' to 'Recent activity', this fails. What's your reasoning?